opengever/maintenance/scripts/repository_migration.py

```diff
@@ -895,6 +959,9 @@ def insert_label_row(self, sheet):
             'Verletzt Leafnode Prinzip',
             'Ist ungultig',
 
+            # metadata
+            'Metadaten Nichtueberstimmung',
```

Comment:
That doesn't quite work in German 😉  I'd say `Metadaten stimmen nicht ueberein`, unless you need a noun-ified message. Then I'd say `Metadaten-Aenderung`.

Reply:
😅